CMPUT_301_Project/app/src/main/java/com/example/cmput_301_project/NavBar.java

```diff
@@ -0,0 +1,81 @@
+package com.example.cmput_301_project;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import com.google.android.material.bottomnavigation.BottomNavigationView;
+
+
+/**
+ * A simple {@link Fragment} subclass.
+ * create an instance of this fragment.
+ */
+public class NavBar extends Fragment {
+
+
+
+    public NavBar() {
+        // Required empty public constructor
+    }
+
+    /**
+     * @return A new instance of fragment NavBar.
+     */
+    BottomNavigationView bottomNavigationView;
+    //@Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+
+        MainPage mainActivity = (MainPage) getActivity();
+        View view = inflater.inflate(R.layout.fragment_nav_bar, container, false);
+        //Initialize bottomNavigation view
+        bottomNavigationView = view.findViewById(R.id.bottomNavigationView);
+        bottomNavigationView.setSelectedItemId(R.id.home);
+        bottomNavigationView.setOnItemSelectedListener(item -> {
+            String name;
+            //Switch clause that sets alternates text depending on menu item selected
+
+            switch (item.getItemId()) {
+                case R.id.home:
+                    name = "HOME";
+                    mainActivity.textView.setText(name);
```

Comment:
Ideally this should be connected to a new page load to test if the fragment remembers it's state when reloaded, but we can make it work (do not block PR for this)